signac/contrib/job.py

```diff
@@ -73,9 +214,9 @@ class Job:
     """
 
     FN_MANIFEST = "signac_statepoint.json"
```

Comment:
Can we replace "manifest" with "state point" universally? (Depends on whether we consider this constant to be internal API, which it probably should be.) Maybe a topic for another PR / issue...?
```suggestion
    _FN_STATE_POINT = "signac_statepoint.json"
```

Reply:
Yes, I'd consider all the class-level constants to currently be part of the API (and they even have docstrings). Let's discuss the possibility of changing this in a separate issue.